src/permission.js

```diff
@@ -252,6 +270,14 @@ export function preparePermissionsForm(permissions: string[]) {
         }
       }
     },
+    groups: {
+      classNames: "field-groups",
+      ...groups.reduce((acc, group) => {
+        const {id: gid} = group;
+        acc[gid] = {"ui:widget": "checkboxes"};
```

Comment:
Nit: I believe `acc[group.id] = {"ui:widget": "checkboxes"};` would be okay.

Also, you can return everything immediately:

```js
return {...acc, [group.id]: {"ui:widget": "checkboxes"}};
```